services/blockchain-connector/events/proxy/allEvents.js

```diff
@@ -37,12 +36,12 @@ const exportAllEvents = async () => {
 	await subscribeToAllRegisteredEvents();
 
 	const registeredEvents = await getRegisteredEvents();
-	const allMethods = registeredEvents.map(event => {
+	const allEvents = events.concat(registeredEvents);
```

Comment:
This is not generic enough. We'll need it to be dynamic so that all the future custom events are also automatically exported.